docs/release-notes/det-deploy-clean-orphaned.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Breaking Changes**
```

Comment:
I thought that's an improvement!

Reply:
Heh, I changed the section, but I put in an inline breaking change marker, since this will change the behavior for people doing the same thing before and after this release and _maybe_ someone cares about the old thing.